bugbug/phabricator.py

```diff
@@ -146,9 +147,9 @@ def download_revisions(rev_ids: Collection[int]) -> None:
 def download_modified_revisions():
     try:
         last_modified = db.last_modified(REVISIONS_DB)
-    except Exception as e:
-        if str(e) == "Last-Modified is not available":
-            return
+    except LastModifiedNotAvailable as e:
+        logger.error("Exception:", e)
```

Comment:
Let's keep the same behavior as before: no logging

Reply:
excluded logging